account-compression/sdk/src/merkle-tree/index.ts

```diff
@@ -53,6 +59,30 @@ export class MerkleTree {
     this.depth = nodes.peek()!.level + 1;
   }
 
+  /**
+   * This is the recommended way to create MerkleTrees.
+   * If you're trying to match an on-chain MerkleTree,
+   * set `depth` to `{@link ConcurrentMerkleTreeAccount}.getMaxDepth()`
+   *
+   * @param leaves leaves of the tree
+   * @param depth number of levels in the tree
+   * @returns MerkleTree
+   */
+  static sparseMerkleTreeFromLeaves(
+    leaves: Buffer[],
+    depth: number
+  ): MerkleTree {
+    const _leaves: Buffer[] = [];
```

Comment:
Do you need this at all? Or do you mean to pass this in when creating the tree to avoid modifying the user's `leaves` array? That would mean you'll need to push the empty buffers to this too

Reply:
Meant to pass `_leaves` to ctor in order to leave user's array untouched. Thanks for catching!